manifests/profiles/default.yaml

```diff
@@ -292,8 +287,6 @@ spec:
           maxNumberOfAttributes: 200
           maxNumberOfAnnotations: 200
           maxNumberOfMessageEvents: 200
-      mtls:
```

Comment:
What effect does this flag have on mtls? I am a little concerned we are moving too fast on variable renaming and may break core fundamental functionality without properly documenting. This would be one of those assignments I have concern with (also, accesslog  and associated friends).

Reply:
mtls.enabled literally does nothing currently as of 1.5 or 1.6. In 1.6 we have a deprecation message if its used, along with accesslog and the others mentioned here